changes/tasks.py

```diff
@@ -524,6 +560,17 @@ def check_momconnect_change_language(self, data_fields, change):
         else:
             return []
 
+    def check_momconnect_change_msisdn(self, data_fields, change):
+        """
+        Ensures that a new msisdn is provided, and that it is a valid msisdn.
+        """
+        if "msisdn" not in data_fields:
```

Comment:
hmmm good point. It would be nice to keep the knowledge of which number was the default but I'm not sure how to do that. Perhaps we could populate a "previous_default" field?
another thought was we should probably record the msisdn that the change request came from, although I don't think that's specific to the msisdn change.